v3-nginx/docker-entrypoint.d/91-modsecurity-envsubst.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
I would generalize these two functions into a single function, accepting arguments (now it's nearly 100% duplicated code). Then the two entrypoint scripts would simple call that function.

Reply:
This came from upstream, but removed in new iteration.